<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>状态栏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_2"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveSomewhere"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关联变量分析</string>
    </property>
    <addaction name="actionShortDistanceAnalysis"/>
    <addaction name="actionDependenceAnalysis"/>
    <addaction name="actionBrotherAnalysis"/>
    <addaction name="actionSimilarAnalysis"/>
    <addaction name="action_IOAnalysis"/>
    <addaction name="actionAll"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="actionHeadFiles"/>
    <addaction name="action_IR"/>
    <addaction name="actionBC"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>代码格式整理</string>
    </property>
    <addaction name="actionLLVMStyle"/>
    <addaction name="actionGoogleStyle"/>
    <addaction name="actionChromiumStyle"/>
    <addaction name="actionMozillaStyle"/>
    <addaction name="actionWebKitStyle"/>
    <addaction name="actionMicrosoftStyle"/>
    <addaction name="actionGNUStyle"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDependenceAnalysis">
   <property name="text">
    <string>数据依赖、控制依赖分析</string>
   </property>
   <property name="toolTip">
    <string>数据依赖、控制依赖分析</string>
   </property>
  </action>
  <action name="actionShortDistanceAnalysis">
   <property name="text">
    <string>较近访问距离分析</string>
   </property>
   <property name="toolTip">
    <string>较近访问距离分析</string>
   </property>
  </action>
  <action name="actionSimilarAnalysis">
   <property name="text">
    <string>相似名称分析</string>
   </property>
  </action>
  <action name="action_IOAnalysis">
   <property name="text">
    <string>连续IO地址分析</string>
   </property>
   <property name="toolTip">
    <string>连续IO地址分析</string>
   </property>
  </action>
  <action name="actionBrotherAnalysis">
   <property name="text">
    <string>兄弟元素分析</string>
   </property>
   <property name="toolTip">
    <string>兄弟元素分析</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>综合分析</string>
   </property>
   <property name="toolTip">
    <string>综合分析</string>
   </property>
  </action>
  <action name="actionSaveSomewhere">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_IR">
   <property name="text">
    <string>生成程序IR</string>
   </property>
   <property name="toolTip">
    <string>生成程序IR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionBC">
   <property name="text">
    <string>生成程序字节码</string>
   </property>
   <property name="toolTip">
    <string>生成程序字节码</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionHeadFiles">
   <property name="text">
    <string>设置头文件的路径</string>
   </property>
   <property name="toolTip">
    <string>设置头文件的路径</string>
   </property>
  </action>
  <action name="actionLLVMStyle">
   <property name="text">
    <string>LLVMStyle</string>
   </property>
  </action>
  <action name="actionGoogleStyle">
   <property name="text">
    <string>GoogleStyle</string>
   </property>
  </action>
  <action name="actionChromiumStyle">
   <property name="text">
    <string>ChromiumStyle</string>
   </property>
  </action>
  <action name="actionMozillaStyle">
   <property name="text">
    <string>MozillaStyle</string>
   </property>
  </action>
  <action name="actionWebKitStyle">
   <property name="text">
    <string>WebKitStyle</string>
   </property>
  </action>
  <action name="actionMicrosoftStyle">
   <property name="text">
    <string>MicrosoftStyle</string>
   </property>
  </action>
  <action name="actionGNUStyle">
   <property name="text">
    <string>GNUStyle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
